src/main/java/com/dft/theiconic/OrdersApi.java

```diff
@@ -36,6 +35,19 @@ public OrdersResponse getOrders(HashMap<String, String> parameters) {
         return getRequestWrapped(request, handler);
     }
 
+    public OrdersResponse getOrders(String orderId) {
```

Comment:
Rename method name getOrders to getOrderById

Reply:
yes sir